borg/archiver.py

```diff
@@ -1213,9 +1213,12 @@ def build_parser(self, args=None, prog=None):
             be specified for ARCHIVE1.
 
             For archives created with Borg 1.1 or newer only chunks IDs
-            are compared, which is very fast. For archives prior to Borg 1.1 chunk contents must be
-            compared, unless you are sure they were created with the same chunker params
-            (pass --same-chunker-params). Note that the chunker params changed from Borg 0.xx to 1.0.
+            are compared, which is very fast.
```

Comment:
you still should say that it is only doing that as long as the chunker params are the same.